hopsworks-common/src/main/java/io/hops/hopsworks/common/hdfs/DistributedFileSystemOps.java

```diff
@@ -257,6 +260,82 @@ public FileStatus getFileStatus(Path location) throws IOException {
     return dfs.getFileStatus(location);
   }
 
+  /**
+   * Attach XAttr to file or directory in the given path
+   *
+   * @param path
+   * @return
+   * @throws IOException
+   */
+  public void setXAttr(String path, String name, byte[] value, EnumSet<XAttrSetFlag> flag) throws IOException {
```

Comment:
Can you add the `Path path` equivalent?

Reply:
Already there
https://github.com/AlexHopsworks/hopsworks/blob/provenance/hopsworks-common/src/main/java/io/hops/hopsworks/common/hdfs/DistributedFileSystemOps.java#L821